### PR TITLE
Re-write getting started docs

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1233 

#### Overview 

We've gotten lots of feedback on our getting started docs. Here are some takeaways

* beginners need a simpler *focused* getting started guide
* beginners need more help with certain gotchas along the process
* beginners could benefit from screenshots / gifs
* beginners can benefit by having common questions answered (i.e. why yarn)
* beginners don't always feel ready to jump into an up for grab. Having a good first activity is helpful
* advanced users will figure it out. They're a minority compared to the many beginners who are good devs, but have questions
* the docs are currently spread out and have too much information that comes before the setup instructions. they can always learn the local dev stuff after they're setup.

#### Summary of Changes

1. add getting started doc 
2. add first activity doc
3. consolidate contributing doc
4. consolidate readme

#### Test Plan

- [x] we should double check content to make sure it makes sense
- [x] we should click all the links

CC @Garbee, @clarkbw, @viswanathamsantosh

### CODE REVIEW

File: docs/debugging-the-debugger.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+1. When you inspect the running debugger app, you're debugging a web app
+2. The Debugger like other applications has an API to communicate with the browser
+3. There's no magic here. If you can build a web app, you can hack on the debugger!
+4. You are the debugger's principal customer. Remember, the customer is always right!
+
+
+**Share what you know** Give a talk in school, work, or a local meetup. I'm willing to bet your audience will not know the debugger is a web app!
+
+- here are @amitzur's [slides](https://docs.google.com/presentation/d/1jdnvL-BwwxEuFbb9tiRxcT6UT-Ua0jGhy9FKBT4b43E/edit) from his [talk](https://twitter.com/amitzur/status/790153843946426369)
```
Comment (jasonLaster) [inline docs/debugging-the-debugger.md]: @clarkbw i think your slides could be good here too

File: docs/getting-setup.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+Close firefox and re-open it with the `firefox-bin` command.
+
+
+#### Starting Chrome
+
+There are two ways to run chrome. Here's the easy way to run chrome
+
+```bash
+yarn run chrome
+```
```
Comment (jasonLaster) [inline docs/getting-setup.md]: 🌟 @Garbee  I think you'll like this!

File: docs/getting-setup.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+#### Windows + Linux setup
+
+Windows and Linux should *just work*, but unfortunately there are several edge cases.
+
+If you find any issues on these two platforms comment on these issues:
+* [windows](https://github.com/devtools-html/debugger.html/issues/1248)
+* [linux](https://github.com/devtools-html/debugger.html/issues/1082)
+
+
+```
```
Comment (jasonLaster) [inline docs/getting-setup.md]: ughh, meant to have some context here. not sure if we need the same linux details

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# debugger.html
+# Debugger.html
```
Comment (clarkbw) [inline README.md]: Why the caps?  We've always gone lowercase.

File: bin/chrome-driver.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+const spawn = require('child_process').spawn;
+
+const chromeBinary = "/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome"
```
Comment (clarkbw) [inline bin/chrome-driver.js]: Maybe a note or file a followup issue to get Windows working.

File: docs/getting-setup.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+
+### Appendix
+
+#### Quick Setup
+
+This setup is for people on the DevTools team and DevTools wizards.
+
+```bash
+npm i -g yarn@0.16.1
+g clone git@github.com:devtools-html/debugger.html.git
```
Comment (clarkbw) [inline docs/getting-setup.md]: `git clone`

File: docs/getting-setup.md [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+/Applications/Firefox.app/Contents/MacOS/firefox-bin --start-debugger-server 6080 -P development
+
+# create a new terminal tab
+cd debugger.html
+yarn start
+```
+
+#### Starting Firefox
+
+If you're looking for an alternative to `yarn run firefox`, you have two options: cli, gcli.
```
Comment (clarkbw) [inline docs/getting-setup.md]: I think you might have skipped the **GCLI** option below.

File: README.md (hunk 4 above)
Comment (jasonLaster) [inline README.md]: haha - yeah, wanted to have a conversation about and hear why. sorry forgot to comment

File: bin/chrome-driver.js (hunk 5 above)
Comment (jasonLaster) [inline bin/chrome-driver.js]: good point

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -35,126 +25,19 @@ We respect your time and want to help you make the most of it as you learn more
   * [Configuration](#configuration)
     * [Create a local config file](#create-a-local-config-file)
   * [Issues and Pull Request labels](#issues-and-pull-requests)
+[Project Overview](#sd)
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
```
Comment (jasonLaster) [inline CONTRIBUTING.md]: not necessary

**Pull request merged**